### PR TITLE
footer: some changes

### PR DESCRIPTION
**Pull request by amarts**
* copyright to the registered company
* link to linkedin company profile
* Move 'Twitter' and 'Blog' out of community section
* Change link to Contributors and add 'Projects' link.
* Add 'Write to us' Email contact

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -9,6 +9,9 @@ <h2 class="text-xl text-yellow-600">Get Started</h2>
                 <a class="footer-menu" href="{{ "/docs/get-involved" | relative_url }}">
                     <i class="fas fa-users pr-2"></i>Get Involved
                 </a>
+                <a class="footer-menu" href="https://linkedin.com/company/{{ site.linkedin }}">
+                    <i class="fab fa-twitter pr-2"></i>LinkedIn
```
Comment (aravindavk) [inline _includes/footer.html]: `fa-linkedin` Font awesome icon for LinkedIn

File: _includes/footer.html [hunk 2]
```diff
@@ -32,7 +35,7 @@ <h2 class="text-xl text-yellow-600">Community</h2>
                 </a>
             </div>
             <p class="text-center w-full mt-10">
-                &copy; 2019 <strong>Kadalu Authors</strong>. All rights reserved.
+                &copy; 2020 <strong>Kadalu Software Pvt Ltd.</strong>. All rights reserved.
```
Comment (aravindavk) [inline _includes/footer.html:44]: 2019-2020?

File: _includes/footer.html (hunk 1 above)
Comment (aravindavk) [reply _includes/footer.html]: https://fontawesome.com/icons/linkedin

File: _includes/footer.html (hunk 2 above)
Comment (amarts) [reply _includes/footer.html:44]: I have noticed that on the website, people keep only latest year. At least on many of the company sites.

Comment (aravindavk) [reply _includes/footer.html:44]: ack

File: _includes/footer.html (hunk 1 above)
Comment (amarts) [reply _includes/footer.html]: Done

Comment (aravindavk) [reply _includes/footer.html]: Looks like Twitter and Linkedin icons interchanged

**Review by aravindavk — APPROVED** [review]

**Pull request merged**